packages/api/src/schema/schema.yaml

```diff
@@ -762,6 +762,9 @@ components:
         pub:
           type: string
           description: Base64 of Public key used for JWT verification
+        accessKey:
```

Comment:
Missing changes:
 - Creating an `accessKey` enum value for the `type` field. Notice this is the "type" **for catalyst**, which is why it should NOT be `webhook`. Catalyst has no idea what the access key will be used for.
 - Creating a `playbackId` field (instead of `stream` which doesn't make sense anymore)

Ideally, we should have this object as `oneOf` to have clear & properly validated schemas here. Otherwise you should add at least 1 more check in the code which is checking the `req.body.type` as well. I made a separate comment about that in the code, but I prefer the `oneOf` approach TBH, we shouldn't be writing any schema validation code ourselves, our logic is already complex enough kkkk.